kubeinstallutils/crd.go

```diff
@@ -3,21 +3,22 @@ package kubeinstallutils
 import (
 	"context"
 
+	apiv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+
 	"github.com/rotisserie/eris"
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	apiexts "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func CrdsFromManifest(crdManifestYaml string) ([]*v1beta1.CustomResourceDefinition, error) {
-	var crds []*v1beta1.CustomResourceDefinition
+func CrdsFromManifest(crdManifestYaml string) ([]*apiv1.CustomResourceDefinition, error) {
```

Comment:
Given that we're changing an exported interface of this library, should this be a breaking change?